tests/tck-build-logic/src/main/groovy/org.graalvm.internal.tck-harness.gradle

```
@@ -86,8 +86,10 @@ if (project.hasProperty("baseCommit")) {
 }
 
 def matrixDefault = [
-        "version"     : ["22.3.0", "dev"], // TODO: Add other supported versions
-        "java-version": ["17"], // TODO: Add "17", "19"
+        "graal-java-versions" : [
+//                [ graal: "dev", java: "dev" ], // TODO: uncomment this once we are ready to run tests on JDK19
+                [ graal: "latest", java: "17"]
+        ],
```

I think it makes sense to simplify this a bit:

```suggestion
        "versions" : [
//                [ "graalvm": "dev", "java": "dev" ], // TODO: uncomment this once we are ready to run tests on JDK19
                [ "graalvm": "latest", "java": "17"]
        ],
```

---

I will do this change locally, since this requires one more change.